synfig-studio/images/CMakeLists.txt

```diff
@@ -192,6 +192,8 @@ set(ICONS
 	set_outline_color
 	show_grid_icon
 	show_guideline_icon
+	bound_button_icon_lower
+	bound_button_icon_upper
```

Comment:
Change the icon names to:
- lower_bound_button_icon
- upper_bound_button_icon

@ice0: "button", "mark" or "handle"?

Reply:
true "lower_bound_button_icon" sounds more like an actual phrase, I have a weird habit of making then names have a weird order :".

p.s. will keep it "button" for now and if decided one of the others is better will change then.

Reply:
I think the handle is more correct, because we don't click on it, we drag it.